pkg/iauthnzimpl/consts.go

```diff
@@ -115,6 +122,7 @@ var (
 	qNameRoleUntillPaymentsManager     = appdef.NewQName(airPackage, "UntillPaymentsManager")
 	qNameRoleBOReader                  = appdef.NewQName(airPackage, "BOReader")
 	qNameResellerPortalDashboardViewer = appdef.NewQName(airPackage, "ResellerPortalDashboardViewer")
+	qNameSysGuest                      = appdef.NewQName(appdef.SysPackage, "Guest")
```

Comment:
eliminate, use `istructs.SysGuestLogin` 
note: it is not a QName, it is a login string